src/main/kotlin/org/rust/ide/inspections/fixes/RemoveTypeArguments.kt

```diff
@@ -32,8 +32,6 @@ class RemoveTypeArguments(
     }
 
     private fun RsTypeArgumentList.removeTypeParameters() {
-        if (lifetimeList.size > 0) return
```

Comment:
Unfortunately, without this condition, next line breaks code when the declaration doesn't have type parameter.
![2021-01-10 21 42 52](https://user-images.githubusercontent.com/2539310/104132207-cd4f8680-538c-11eb-899c-bcd2dbe1c3da.gif)

Reply:
Also, don't forget about const generics.
For example, in the following case
![2021-01-10 21 54 09](https://user-images.githubusercontent.com/2539310/104132459-616e1d80-538e-11eb-96c2-a554a51a438b.gif)
the quick-fix probably shouldn't be suggested at all because it's not clear what type argument it should remove.

But it seems, all our checks and quick fixes don't take into account const arguments so I suppose it can be supported in separate PR